homeassistant/components/simplepush/notify.py

```diff
@@ -61,11 +61,26 @@ def send_message(self, message: str, **kwargs: Any) -> None:
         """Send a message to a Simplepush user."""
         title = kwargs.get(ATTR_TITLE, ATTR_TITLE_DEFAULT)
 
+        attachments = None
         # event can now be passed in the service data
         event = None
         if data := kwargs.get(ATTR_DATA):
             event = data.get(ATTR_EVENT)
 
+            attachments_data = data.get(ATTR_ATTACHMENTS)
+            if isinstance(attachments_data, list) and attachments_data:
+                attachments = []
+                for attachment in attachments_data:
+                    if "attachment" in attachment and "thumbnail" in attachment:
+                        attachments.append(
+                            {
+                                "video": attachment["attachment"],
+                                "thumbnail": attachment["thumbnail"],
+                            }
+                        )
+                    elif "attachment" in attachment:
+                        attachments.append(attachment["attachment"])
```

Comment:
Can't we have the user mention "video" and "thumbnail" instead of "attachment" and "thumbnail" when adding a video under "attachments"? This way you can directly use the "attachments" key.
If I am not mistaken the "attachments" key entered by the user would be like this:
```yaml
attachments:
    - https://upload.wikimedia.org/wikipedia/commons/e/ee/Sample_abc.jpg
    - video: http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ElephantsDream.mp4
      thumbnail: http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/images/ForBiggerEscapes.jpg

Reply:
@engrbm87, thanks a lot for your review!

Your suggestion makes a lot of sense to me.

Reply:
@engrbm87 I implemented your suggestion in commit 9bc18c3e2d58eb618d594d0de244e763860200c3.

Reply:
Added a comment based on the last commit